src/components/messenger/list/start-group-panel/index.tsx

```diff
@@ -67,7 +67,11 @@ export class StartGroupPanel extends React.Component<Properties, State> {
       <>
         <PanelHeader title='Select members' onBack={this.props.onBack} />
         <div {...cn('search')}>
-          <AutocompleteMembers search={this.props.searchUsers} onSelect={this.selectOption}>
+          <AutocompleteMembers
+            search={this.props.searchUsers}
+            onSelect={this.selectOption}
+            selectedOptions={this.state.selectedOptions}
```

Comment:
I can't decide if this would feel more like a common component if we used `value` instead of `selectedOptions`. What do you think?

Reply:
Yeah from my perspective if we directly use `value` here then it wouldn't feel much like a common component, it may be too specific 🤔

Reply:
Basically all standard HTML input components use `value`. That's why it could make sense to use that. It doesn't matter, I don't think. Was just thinking out loud.